app/containers/CalendarModal.js

```diff
@@ -0,0 +1,171 @@
+import React, {Component} from 'react';
+import {
+  Modal,
+  DatePickerIOS,
```

Comment:
Why do we have a date picker? isn't the modal just gonna have read only information about the shift?